src/components/login/signin.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { FormField, TextInput, Label, InlineAlert, Button } from 'evergreen-ui';
+import Box from 'ui-box';
+import logo from '../logo/AgroeasyLogo.png';
+
+export default class Signin extends React.Component {
+  render() {
+    return (
+       
+      <FormField display="flex" alignItems="center" justifyContent="center" style={{marginTop:'70px'}}>
```

Comment:
do not do in-line styles. If you want to style an element, give it a className or id and specify the styles in the css file. This applies to this element and every other element in this Pull Request

Reply:
Strings should be moved to constant.js and imported for use